src/Registry/Registry.php

```diff
@@ -16,19 +16,19 @@ class Registry
     /**
      * List of all fresh resources
      *
-     * @var array
+     * @var array<bool>
      */
     protected array $fresh = [];
 
     /**
      * List of all connections
      *
-     * @var array
+     * @var array<mixed>
```

Comment:
`array<mixed>` is more like `[1, 2, 'f']`. This looks like it should be more like key value pairs like. That type is `array<string, mixed>`

Reply:
addressed